src/data.toml

```diff
@@ -1,18 +1,22 @@
 thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\nYou may use a system information app such as `CPU-Z` or `AIDA64` to check the architecture of your CPU and operating system."
 broken_addons = "You can find a curated list of broken add-ons in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
 altstore_pal = "AltStore PAL is an alternate app marketplace for notarized applications. You cannot use it to sideload.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
 no_passkeys_when_pre_patched = "Passkeys are not supported when using the pre-patched application. Please use a different method to log in to your account or use the LSPosed module / iOS tweak."
-ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev cert instead (which generally uses an explicit App ID by default) if you want this functionality.\n\nKeep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-ios_file_access = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively disables the file access capability.\nYou need to sign the application with a local dev cert instead (which generally uses an explicit App ID by default) if you want this functionality.\n\nKeep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-addon_404 = "You are not supposed to visit the url directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev CERT instead if you want this functionality.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
+addon_404 = "You are not supposed to visit the URL directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+cloud_sync_not_working = "You might be using an outdated version of Cloud Sync.\n\n- Make sure you have the [latest version](<https://bunny.nexpid.xyz/cloud-sync/>) installed.\n- Reinstall the [antied plugin](<https://angelix1.github.io/MP/angel/antied/>) if you use it.\n\nPerform a local backup to avoid losing data permanently."
```

Comment:
```suggestion
cloud_sync_not_working = "You might be using an outdated version of Cloud Sync, as older versions are no longer supported. To resolve this issue, please follow these steps:\n\n1. **Update Cloud Sync**: Ensure you have the latest version of Cloud Sync installed. You can find the updated version in the pins.\n2. **Reinstall `antied`**: If updating Cloud Sync doesn't resolve the issue and you have antied installed, uninstall and then reinstall it from the following link: [antied Installation](https://angelix1.github.io/MP/angel/antied/).\n\n**Important:**\nIt's recommended to use a local backup to avoid accidental erasure of cloud data, as overwritten cloud data cannot be recovered once replaced with an empty sync.
```

Reply:
Its best to keep the response as short as possible so the current auto response seems fine.